app/controllers/SubscriptionController.scala

```diff
@@ -50,25 +53,23 @@ class SubscriptionController @Inject() (
   val config: AppConfig,
   authenticate: AuthAction,
   subscriptionConnector: SubscriptionConnector,
+  auditService: AuditService,
   override val controllerComponents: ControllerComponents
 )(implicit executionContext: ExecutionContext)
     extends BackendController(controllerComponents)
     with Logging {
 
   def createSubscription: Action[JsValue] = authenticate(parse.json).async { implicit request =>
-    val subscriptionSubmissionResult: JsResult[CreateSubscriptionForCBCRequest] =
-      request.body.validate[CreateSubscriptionForCBCRequest]
-
-    subscriptionSubmissionResult.fold(
-      invalid = _ =>
-        Future.successful(
-          BadRequest("CreateSubscriptionForCBCRequest is invalid")
-        ),
-      valid = sub =>
-        for {
-          response <- subscriptionConnector.sendSubscriptionInformation(sub)
-        } yield convertToResult(response)(implicitly[Logger](logger))
-    )
+    request.body
+      .validate[CreateSubscriptionForCBCRequest]
+      .fold(
+        invalid = _ => Future.successful(BadRequest("CreateSubscriptionForCBCRequest is invalid")),
+        valid = subscriptionRequest =>
+          for {
+            response <- subscriptionConnector.sendSubscriptionInformation(subscriptionRequest)
+            _        <- if (response.status == OK) sendAuditEvent(subscriptionRequest, response) else Future.unit
+          } yield convertToResult(response)(implicitly[Logger](logger))
```

Comment:
If we have the following implicit on the top:
```
implicit val logging = logger
```
Then we don't have to explicitly pass these loggers around ?